utils/database.py

```diff
@@ -0,0 +1,23 @@
+import asyncio
+import asyncpg
+
+async def create_pool():
+    pool = await asyncpg.create_pool(dsn='postgresql://guildbot:gu1ldb0t@localhost/guildbot')
+    async with pool.acquire() as conn:
+        await conn.execute("CREATE TABLE IF NOT EXISTS threads (channel_id, author_id)")
+
+class Database():
+    def __init__(self):
+        self.pool = asyncio.get_event_loop().run_until_complete(create_pool)
```

Comment:
The code errors without this
```suggestion
        self.pool = asyncio.get_event_loop().run_until_complete(create_pool())
```

Reply:
Fixed with c362abe927cf7e839171316ce7bd6143994d44d0